### PR TITLE
Redis: updated company name and logo

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

**Review by gkorland — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Thanks for helping!
 
 ## Sponsorship
 
-![RedisLabs Logo](logo-redislabs.png)
+![Redis Logo](redis-logo-full-color-rgb.png)
```
Comment (gkorland) [inline README.md:149]: @sazzad16 we should also add a link to redis.com

Comment (sazzad16) [inline README.md:149]:
```suggestion
[![Redis Logo](redis-logo-full-color-rgb.png)](redis.com)
```

File: README.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ Thanks for helping!
 
 ## Sponsorship
 
-![RedisLabs Logo](logo-redislabs.png)
+[![Redis Logo](redis-logo-full-color-rgb.png)](redis.com)
```
Comment (sazzad16) [inline README.md:149]:
```suggestion
[![Redis Logo](redis-logo-full-color-rgb.png)](https://redis.com/)
```

File: README.md (hunk 1 above)
Comment (sazzad16) [reply README.md]: @gkorland done

**Review by gkorland — APPROVED** [review]

**Pull request merged**